src/components/About/ListItem.module.scss

```diff
@@ -0,0 +1,94 @@
+@import '../../styles/variables';
+@import '../../styles/fonts';
+
+.listItem {
+  flex: 0 0.48 48%;
+  padding-bottom: 50px;
+  cursor: pointer;
```

Comment:
To increase readability you should add blank lines and tabs every time you add a new nesting level in your styles.